### ISSUE TITLE
Add option to profile selected blocks of code inside of procedure.

### ISSUE THREAD

**Issue by rgreat** [ase379/gpprofile2017#121]
I need to find a problem somewhere inside a procedure.

Adding extra named functions would help.

Example: 

```
begin
  // Some Code
  ProfilerEnterBlock('MyCode1&2');
  // Some Code
  ProfilerEnterBlock('MyCode1');
  // Some Code
  ProfilerExitBlock('MyCode1');
  // Some Code
  ProfilerEnterBlock('MyCode2');
  // Some Code
  ProfilerExitBlock('MyCode2');
  // Some Code
  ProfilerExitBlock('MyCode1&2');
  // Some Code
end;
```

**Comment by ase379** [comment]
Hi, this is addressed with the measure points (#92).

**Comment by rgreat** [comment]
You could add a simple example in the readme.
Can't find how to use these.
